### PR TITLE
[Doc] inference

### PR DESCRIPTION
**Pull request by plyfager**

### CODE REVIEW

File: docs/en/user_guides/inference.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Inference
```
Comment (LeoXing1996) [inline docs/en/user_guides/inference.md:1]: Rename this file to `3_inference.md`.

Comment (LeoXing1996) [inline docs/en/user_guides/inference.md:1]: # Tutorial 3: Inference with existing models

File: docs/en/user_guides/inference.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Currently, we support various popular generative models, including unconditional GANs, conditional GANs, image translation models, internal GANs and diffusion models.
+In this section, we will specify how to sample fake images by using our existing models. For model inference, all of the APIs have been included in [mmgen/apis/inference.py](https://github.com/open-mmlab/mmgeneration/tree/1.x/mmgen/apis/inference.py). The most important function is `init_model` for creating a generative model from a config. Then, adopting the sampling function in this file with the generative model will offer you the synthesized images.
```
Comment (LeoXing1996) [inline docs/en/user_guides/inference.md:4]: https://github.com/open-mmlab/mmgeneration/tree/1.x/mmgen/apis/inference.py ->
Can we use a relative link for this url?

File: docs/en/user_guides/inference.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+```
+
+Note that more customized arguments are also offered to customizing your sampling procedure. Please use `python demo/translation_demo.py --help` to check more details.
+
+## Sample images with diffusion models
+
+MMGeneration provides high-level APIs for sampling images with diffusion models. Here is an example for building I-DDPM and obtaining the synthesized images.
+
+```python
+import mmcv
```
Comment (LeoXing1996) [inline docs/en/user_guides/inference.md:123]: All imported `mmcv` seems not used. Can we remove this?

File: docs/en/user_guides/inference.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+# Specify the path to image you want to translate
+image_path = 'tests/data/paired/test/33_AB.jpg'
+device = 'cuda:0'
+# init a generatvie
+model = init_model(config_file, checkpoint_file, device=device)
+# translate a single image
+translated_image = sample_img2img_model(model, image_path, target_domain='photo')
+```
+
+Indeed, we have already provided a more friendly demo script to users. You can use [demo/translation_demo.py](https://github.com/open-mmlab/mmgeneration/tree/1.x/mmgen/demo/translation_demo.py) with the following commands:
```
Comment (LeoXing1996) [inline docs/en/user_guides/inference.md:105]: Relative references

File: docs/en/user_guides/inference.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+checkpoint_file = 'https://download.openmmlab.com/mmgen/stylegan2/stylegan2_c2_ffhq_1024_b4x8_20210407_150045-618c9024.pth'
+
+device = 'cuda:0'
+# init a generatvie
+model = init_model(config_file, checkpoint_file, device=device)
+# sample images
+fake_imgs = sample_unconditional_model(model, 4)
+```
+
+Indeed, we have already provided a more friendly demo script to users. You can use [demo/unconditional_demo.py](https://github.com/open-mmlab/mmgeneration/tree/1.x/mmgen/demo/unconditional_demo.py) with the following commands:
```
Comment (LeoXing1996) [inline docs/en/user_guides/inference.md:26]: relative reference

File: docs/en/user_guides/inference.md (hunk 2 above)
Comment (zengyh1900) [inline docs/en/user_guides/inference.md:4]: 'by using our existing models' --> 'by using our pre-trained models'

File: docs/en/user_guides/inference.md (hunk 1 above)
Comment (zengyh1900) [inline docs/en/user_guides/inference.md:1]: For a long tutorial, it would be better to give an outline at the beginning.

**Pull request merged**